frontend/src/features/public-form/utils/createSubmission.ts

```diff
@@ -284,7 +420,8 @@ const encryptAttachment = async (
     }
 
     return { id, encryptedFile: encodedEncryptedAttachment }
-  } catch (error) {
+  } catch (err) {
```

Comment:
why do we need to do this?

Reply:
this was throwing a type error for me, i don't know how it got through linting :'( i encountered issues without making this type change